CONTRIBUTING.md

```diff
@@ -329,6 +329,73 @@ If you already have the `refinebio-examples` docker image:
 docker run --mount type=bind,target=/home/rstudio,source=$PWD ccdl/refinebio-examples snakemake --cores 4
 ```
 
+### Pull Requests
+
+There are two protected branches in this repo:
+
+`staging` - where changes are initially sent for testing and previewing.
+`master` - where changes that were tested are made user-facing.
+
+This two branch set up allows us to make incremental changes that we can test before directly making our changes "live".
+It also means that pull requests have some extra methodology.
+
+#### Add to the testing branch: merges to staging
+
+After you've prepared the new or altered material for this repository, you can file a pull request to the `staging` branch (the default branch for this repository).
+The `staging` branch does not feed into the published, user-facing material so you can feel free to make your pull requests as iterative and incremental as it is useful.
+Use `squash and merge` for these merges.
+`squash and merge` will be helpful for us if we need to "cherry pick" commits if particular project or example is not yet ready for user-facing but other material is.
+
+Previews for `staging` branch should be review using html preview.
+Go here and navigate to the pertinent pages that you've changed.
+`http://htmlpreview.github.io/?https://github.com/AlexsLemonade/refinebio-examples/gh-pages-stages/01-getting-started/getting-started.html`
+
+#### Make it live: staged changes merged to master
+
+Once a set of changes that are merged to `staging` are ready to be published and made "live", you can file a merge to `master` pull request.
+These types of PRs should only involve well-polished and ready for the public material.
+
+**Scenario 1: All changes from staging should be made live**  
+
+If you are certain that all changes in `staging` should be carried over to `master`, first create a new branch from the most up-to-date `staging` branch.
+Now you can use the new branch to create the pull request to master as you would normally do.
+This allows you can resolve any merge conflicts in your new branch so we don't do a bad thing by committing conflict resolutions directly to `staging`.
+
+**Scenario 2: Only some changes from staging should be made live**  
```

Comment:
I think both scenarios can start with a branch from `master`. In scenario 1 the next step is just to merge `staging`  into the new branch. Scenario 2 is cherry picking commits from staging.  This makes things a bit more uniform and easy to explain: "Branch from `origin/master`, then merge in what you need, which may be everything, then file a PR back to master."

Reply:
Okay. I like that. Will change.